poe-tasks/manifest-only-connector-tasks.toml

```diff
@@ -12,6 +12,46 @@
 
 [tasks]
 
-test-all = "uvx 'airbyte-cdk[dev]' connector test"
+get-language = "echo 'manifest-only'" # Use with -qq to get just the language name
+fetch-secrets = "airbyte-cdk secrets fetch ${POE_PWD}"
```

Comment:
We use `${POE_PWD}` for the cwd. If another directory is the cwd, things might not work as expected.

Note: Running `poe source klaviyo fetch-secrets` from the root of the repo will _also_ set the working directory correctly, so commands like this can still work from the repo root without extra steps.